pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -26,6 +26,12 @@ self: super: {
   # successfully with recent versions of the compiler).
   bin-package-db = null;
 
+  # waiting for release: https://github.com/jwiegley/c2hsc/issues/41
+  c2hsc = appendPatch super.c2hsc (pkgs.fetchpatch {
+    url = "https://github.com/jwiegley/c2hsc/commit/490ecab202e0de7fc995eedf744ad3cb408b53cc.patch";
+    sha256 = "0z0x6jsf7jz32kmjxp077h0ca98hwjyx16kpk9660id7ikmbcz91";
```

Comment:
Hmm, I appear to be getting a different hash for this:

```console
$ nix-build -A haskellPackages.c2hsc --arg config '{allowBroken=true;}'
these derivations will be built:
  /nix/store/b2q7nszrzi629nk1f79jsiya2yv8f1dg-490ecab202e0de7fc995eedf744ad3cb408b53cc.patch.drv
  /nix/store/07llqlbz5b5jjhyxlkwiv82banp4c0dm-c2hsc-0.7.1.drv
building '/nix/store/b2q7nszrzi629nk1f79jsiya2yv8f1dg-490ecab202e0de7fc995eedf744ad3cb408b53cc.patch.drv'...

trying https://github.com/jwiegley/c2hsc/commit/490ecab202e0de7fc995eedf744ad3cb408b53cc.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1797  100  1797    0     0  18525      0 --:--:-- --:--:-- --:--:-- 18525
hash mismatch in fixed-output derivation '/nix/store/ilq40b6a9k2n53ch5pikz39wn4xpfsx9-490ecab202e0de7fc995eedf744ad3cb408b53cc.patch':
  wanted: sha256:0z0x6jsf7jz32kmjxp077h0ca98hwjyx16kpk9660id7ikmbcz91
  got:    sha256:1c7knpvxr7p8c159jkyk6w29653z5yzgjjqj11130bbb8mk9qhq7
cannot build derivation '/nix/store/07llqlbz5b5jjhyxlkwiv82banp4c0dm-c2hsc-0.7.1.drv': 1 dependencies couldn't be built
error: build of '/nix/store/07llqlbz5b5jjhyxlkwiv82banp4c0dm-c2hsc-0.7.1.drv' failed
```

Reply:
not sure what's wrong `nix-build -A haskellPackages.c2hsc --arg config '{allowBroken=true;}' --check` builds corerclty plus
```
nix-prefetch-url https://github.com/jwiegley/c2hsc/commit/490ecab202e0de7fc995eedf744ad3cb408b53cc.patch                                                                           ~/nixpkgs3
path is '/nix/store/d2n0i5i9132pw5ijvg9lirxvnw17y5i7-490ecab202e0de7fc995eedf744ad3cb408b53cc.patch'
0z0x6jsf7jz32kmjxp077h0ca98hwjyx16kpk9660id7ikmbcz91
```